.github/workflows/sonarcloud.yml

```
@@ -7,6 +7,9 @@ on:
   pull_request:
     branches:
       - main
+env:
+  AWS_REGION: ap-northeast-2
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)